### PR TITLE
Clarify the documentation for --jobs

### PR DESCRIPTION
**Pull request by toolmantim**

### CODE REVIEW

**Comment by segiddins** [comment]
:+1:

File: man/bundle-install.ronn [hunk 1]
```diff
@@ -68,7 +68,8 @@ update process below under [CONSERVATIVE UPDATING][].
   to this location.
 
 * `--jobs=[<number>]`:
-  Install gems by starting <number> of workers parallely.
+  The maximum number of parallel download and install jobs to run
+  simultaenously. The default is `1`.
```
Comment (agis-) [inline man/bundle-install.ronn]: typo: "simultaneously"

File: man/bundle-install.ronn [hunk 2]
```diff
@@ -68,7 +68,8 @@ update process below under [CONSERVATIVE UPDATING][].
   to this location.
 
 * `--jobs=[<number>]`:
-  Install gems by starting <number> of workers parallely.
+  The maximum number of parallel download and install jobs to run
```
Comment (agis-) [inline man/bundle-install.ronn]: Isn't "parallel" here redundant with "simultaneously" later on?

**Comment by toolmantim** [comment]
Thanks @segiddins @agis-, I've fixed those two issues and pushed a squashed commit

**Comment by segiddins** [comment]
@homu r+

**Comment by segiddins** [comment]
@homu retry

**Pull request merged**